packages/twenty-website/src/app/layout.css

```diff
@@ -30,8 +30,24 @@ a {
 }
 
 nav.toc {
-  width: 200px;
+  width: 20%;
   position: fixed;
-  top: 100px;
+  top: 125px;
   right: 0;
+}
+
+nav.toc ol{
+    list-style-type: circle;
+}
+
+nav.toc li{
+    height: 32px;
+}
+
+nav.toc a{
+    color: #141414;
+    font-family: 'Inter, sans-serif';
```

Comment:
Sent a comment on Slack for font import.

Tip: I like to use 'Inter, monospace' instead 'Inter, sans-serif'  when developing,  it's easier to spot if it's not properly imported!